scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/soic.yaml

```diff
@@ -646,3 +646,29 @@ SOIC-28W_7.5x18.7mm_P1.27mm:
   pitch: 1.27
   num_pins_x: 0
   num_pins_y: 14
+  
+Infineon_PG-DSO-8-59:
+  custom_name_format: 'Infineon_PG-DSO-8-59' 
+  size_source: 'https://www.infineon.com/cms/en/product/packages/PG-DSO/PG-DSO-8-59'
+  body_size_x: 
```

Comment:
Would you mind to try just the nominal and minimum, leaving off max. Or `tolerance" [0, -0.2]`. I can't recall if both or just one way works.